modules/ROOT/pages/con_pipeline-image-resources.adoc

```diff
@@ -0,0 +1,91 @@
+[]
+= Image Resource
+
+An Image resource represents an image that lives in a remote repository. It is
+usually used as [a `Task` `output`] for `Tasks` that build
+images. This allows the same `Tasks` to be used to generically push to any
+registry.
```

Comment:
"This allows the same `Tasks`"--> The IBM Style Guide advises against using the word "allow" in this manner. We could rather say: "With the image resources, you can use the  same `Tasks`  to generically push to any registry.